### PR TITLE
 templates.yml Ultraman

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

**Comment by malonnnn** [comment]
We need to have a talk about who this character is.

The only thing I can find is some weird japanese samurai guy...is that the dude?

If so, his name should match the same colors as his outfit.

**Comment by malonnnn** [comment]
you modified line 2400, unmodify it
https://github.com/malonnnn/ServerConfig/pull/1164/files

Rename "templates.yml Ultraman(E3)" to "templates.yml"

**Comment by malonnnn** [comment]
you changed deathdc, superman, and aquaman.

please unchange them :D

File: Citizens/templates.yml [hunk 1]
```diff
@@ -2301,7 +2301,7 @@ hestia:
     uuid: c555d316-518f-4191-b74b-03c476615df9
     traitnames: spawned,type,inventory,owner,gravity,location
     name: Hestia
-   aquaman:   
+ aquaman:
```
Comment (Supertre03) [inline Citizens/templates.yml]: testing

**Pull request merged**